docs/tutorials/datasets/mhp_v1.py

```diff
@@ -0,0 +1,92 @@
+"""Prepare Multi-Human Parsing V1 dataset.
+========================
+
+`Multi-Human Parsing V1 (MHP-v1) <https://github.com/ZhaoJ9014/Multi-Human-Parsing/>`_ is a human-centric containing
+5 thousands images annotated with 18 object categories.
+This tutorial help you to download MHP-v1 and set it up for later experiments.
+
+.. image:: https://github.com/ZhaoJ9014/Multi-Human-Parsing_MHP/blob/master/Figures/Fig1.png
+   :width: 600 px
+
+.. hint::
+
+   You need 850 MB free disk space to download and extract this dataset.
+   SSD harddrives are recommended for faster speed.
+   The time it takes to prepare the dataset depends on your Internet connection
+   and disk speed. For example, it takes around 1 mins on an AWS EC2 instance
+   with EBS.
+
+Prepare the dataset
+-------------------
+
+We will download and unzip the following files:
+
++-------------------------------------------------------------------------------------------------------+--------+
+| File name                                                                                             | Size   |
++=======================================================================================================+========+
+| `LV-MHP-v1.zip <https://drive.google.com/uc?id=1hTS8QJBuGdcppFAr_bvW2tsD9hW_ptr5&export=download>`_  | 850 MB |
++-------------------------------------------------------------------------------------------------------+--------+
+
+
+The easiest way is to run this script:
+
+ :download:`Download script: mhp_v1.py<../../../scripts/datasets/mhp_v1.py>`
+
+.. code-block:: bash
+
+   python mhp_v1.py
+
+If you have already downloaded the above files and unzipped them,
+you can set the folder name through ``--download-dir`` to avoid
+downloading them again. For example
+
+.. code-block:: python
+
+   python mhp_v1.py --download-dir ~/mhp_v1_downloads
```

Comment:
Change `~/mhp_v1_downloads` to `~/.mxnet/datasets/mhp` because we want users to know where should be the default location to save the datasets.